src/pages/governance/_components/VotingResult.tsx

```diff
@@ -19,6 +25,7 @@ export function VotingResult({
   userSelectedVote,
   setIsChangeVoteClicked,
   isLoading,
+  proposal,
```

Comment:
let's update the variable naming at line 19, 20, 21 to include `Votes`, e.g.: 'yesVotes'